### ISSUE TITLE
[Feature Request] Improve GPX Route Visualization with End Markers and Route Name Display

### ISSUE THREAD

**Issue by dandud100** [organicmaps/organicmaps#5603]
**Is your feature request related to a problem? Please describe.**
Currently, there is no clear way to identify the start and end points of a .gpx route in Organic Maps. This lack of visual indication makes it challenging for users to quickly determine where a route begins and ends, leading to frustration and confusion.

**Describe the solution you would like**
I would like to propose the addition of a visual marker, such as a flag or a similar symbol, at the end of each .gpx route displayed on the map. This will provide users with a clear and intuitive way to identify the route's starting and ending points. Additionally, it would be helpful to have the name of the gpx track displayed above or next to the marker for easy reference.

**Describe alternatives you have considered**
Currently, without this feature in Organic Maps, users may try to infer the start and end points by zooming in and analyzing the track closely. However, this process is time-consuming and not user-friendly, especially for longer and more complex routes.

**Comment by dandud100** [comment]
![tomtom-go-mobile-aktuellen-baustellen-219285](https://github.com/organicmaps/organicmaps/assets/43549788/63f9f458-91bf-44ba-8a8d-a30dc58c29f7)
Maybe we could do this like the red lines of this screenshot from TomTom. And just do the lines like: >===>

**Comment by biodranik** [comment]
@dandud100 that doesn't look clear or convenient to use...

**Comment by dandud100** [comment]
> @dandud100 that doesn't look clear or convenient to use...

Yeah on the provided Screenshot I agree but I don't think one user will create so many tracks so closely so I think it would be a simple solution for now. Do you have a better idea?

**Comment by dandud100** [comment]
Hello, I want to ask if there is any update on this issue? :)

**Comment by biodranik** [comment]
One user can create a lot of tracks when we implement track recorder.
